cmake/RunInputFileTest.sh

```diff
@@ -39,13 +52,17 @@ for expected_code in $4 ; do
     fi
 done
 
+echo "$5"
```

Comment:
Remove? Or label what the value is

Reply:
Yea, removed. That was just debugging stuff :)